frontend/src/app/components/modal/instances/assign-officer-modal.tsx

```diff
@@ -34,7 +34,7 @@ export const AssignOfficerModal: FC<AssignOfficerModalProps> = ({ close, submit,
   const [searchInput, setSearchInput] = useState<string>("");
 
   const officersJson = useAppSelector(selectOfficersByZoneAndAgency(modalData?.agency_code, zone));
-  const searchResults = useAppSelector(searchOfficers(searchInput));
+  const searchResults = useAppSelector(searchOfficers(searchInput, zone));
```

Comment:
This doesn't appear to be working correctly as I can see COSHQ users in the Assign Modal (both list and details view)

Reply:
The dev environment for my PR in OpenShift has only two COS HQ users: MITO and AGUPTA.  They only appear in the officer assignment modal if the complaint is from South Island Zone. For example complaints [24-900002](https://nr-compliance-enforcement-481-frontend.apps.silver.devops.gov.bc.ca/complaint/HWCR/24-900002) or [24-900001](https://nr-compliance-enforcement-481-frontend.apps.silver.devops.gov.bc.ca/complaint/HWCR/24-900001). This is how it should be according to the ticket’s item 2.
If you open the modal for complaints from any other zone COS HQ does not show: [23-032494](https://nr-compliance-enforcement-481-frontend.apps.silver.devops.gov.bc.ca/complaint/HWCR/23-032494) or [23-031744](https://nr-compliance-enforcement-481-frontend.apps.silver.devops.gov.bc.ca/complaint/HWCR/23-031744). 
If you are testing same build and your experience is different could it be caching issue in a browser or am I missing something?

Reply:
@dmitri-korin-bcps  this was a mistake in the ticket.   there was a 'not' missing.